show/main.py

```diff
@@ -1356,16 +1356,29 @@ def show_run_snmp(db, ctx):
 @runningconfiguration.command()
 @click.option('--verbose', is_flag=True, help="Enable verbose output")
 def syslog(verbose):
-    """Show Syslog running configuration"""
+    """Show Syslog running configuration
+    To match below cases:
+    *.* @IPv4:port
+    *.* @[IPv4]:port
+    *.* @[IPv6]:port
+    """
     syslog_servers = []
     syslog_dict = {}
+    re_ipv4_1 = re.compile(r'^\*\.\* @(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}):\d+')
```

Comment:
>^\*\.\* @(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}):\d+ [](http://example.com/codeflow?start=29&length=49)

Thanks for being strict when writing a regex!
However since it is a `show` command, and the data is from a file in filesystem, should we be more relax and do not check for a valid ipv4/v6 address?

we can delegate the check to `rsyslogd -N1`. If anything wrong, we report an empty syslog running config.

Reply:
Hi Qi, according to [rsyslog IPv6 support](https://github.com/Azure/sonic-buildimage/pull/3754/files), I think we can just consider the cases of    ` *.* @[IPv4]:port`  and `*.* @[IPv6]:port` which are all included in square bracket.
This way, I think the initial commit in this PR which checks ending ']' is enough.